content_schemas/formats/homepage.jsonnet

```diff
@@ -6,5 +6,7 @@
       properties: {},
     },
   },
-  links: {},
+  links: {
+    popular_links: "Home page popular links"
```

Comment:
I mentioned below that I think `link_collection` might be a better name for the document type here, _but_ that doesn't mean you can't call the link from the homepage to the link_collection `popular_links`.

Consider [ordered_cabinet_ministers](https://github.com/alphagov/publishing-api/blob/a74f0a509020a20d460f45c07d3227d71d919605/content_schemas/formats/ministers_index.jsonnet#L20) on the ministers index page - the document type it's linking to is going to be Person, but the name of the link is `ordered_cabinet_ministers`. I think that's the right way to represent the semantics of the thing you're linking to.

So I would call the document type `link_collection`, but I would call this link type `popular_links`.

Reply:
Accepted